kuma/users/forms.py

```diff
@@ -204,17 +204,17 @@ def save(self, request):
 
 
 class UserDeleteForm(forms.Form):
+    def __init__(self, *args, username=None, **kwargs):
+        super().__init__(*args, **kwargs)
 
-    ATTRIBUTIONS_KEEP = "keep"
-    ATTRIBUTIONS_DONATE = "donate"
-    ATTRIBUTIONS_CHOICES = (
-        (ATTRIBUTIONS_KEEP, _("Keep my attribution for my page changes")),
-        (
-            ATTRIBUTIONS_DONATE,
-            _('Switch my attribution for my page changes to "Anonymous"'),
-        ),
-    )
+        choices = {
+            "keep": _("Keep my attribution for my page changes"),
+            "donate": _('Switch my attribution for my page changes to "Anonymous"'),
+        }
 
-    attributions = forms.ChoiceField(
-        required=True, choices=ATTRIBUTIONS_CHOICES, widget=forms.widgets.RadioSelect()
-    )
+        if username:
+            choices["keep"] += f" ({username})"
```

Comment:
This reifies the `ugettext_lazy` instance into a real string. Not sure if that's problematic or not; I'm a bit unfamiliar with localization in Django.

Reply:
Perhaps more ideally, the username would be a token in the localized string, since just slamming it onto the end here might not be correct in, say, RTL locales. But it should still be intelligible regardless. E.g.,:

> (callahad) Keep my attribution for my page changes

Reply:
@callahad Yes, the best practice is to include the `username` as a placeholder in the localized string (so translators can reorder the placeholder as befits their language). I'll suggest a change.

Reply:
Other strings we use have `%s` and similar placeholders in them. Maybe use one here?